lib/cartopy/crs.py

```diff
@@ -1930,17 +1930,59 @@ def transform_points(self, src_crs, x, y, z=None):
 
 
 class InterruptedGoodeHomolosine(Projection):
-    def __init__(self, central_longitude=0, globe=None):
-        proj4_params = [('proj', 'igh'), ('lon_0', central_longitude)]
-        super().__init__(proj4_params, globe=globe)
+    """
+    Composite equal-area projection empahsizing either land or
+    ocean features.
+
+    Original Reference:
+    Goode, J. P., 1925: The Homolosine Projection: A new device for
+        portraying the Earth's surface entire. Annals of the
+        Association of American Geographers, 15:3, 119-125,
+        DOI: 10.1080/00045602509356949
+
+    A central_longitude value of -160 is recommended for the oceanic view.
+
+    """
+    def __init__(self, central_longitude=0, globe=None, emphasis='land'):
+        """
+        Parameters
+        ----------
+        central_longitude: optional
```

Comment:
```suggestion
        central_longitude : float, optional
```
All of these should have a space after the name, then the type, followed by optional. See here for the parameter spec: https://numpydoc.readthedocs.io/en/latest/format.html#parameters